RecoPixelVertexing/PixelVertexFinding/plugins/PixelVertexProducerFromSoA.cc

```diff
@@ -127,6 +129,7 @@ void PixelVertexProducerFromSoA::produce(edm::StreamID streamID, edm::Event &iEv
         edm::LogWarning("PixelVertexProducer") << "oops track " << it << " does not exists on CPU " << k;
         continue;
       }
+      auto trackCollection = iEvent.getHandle(tokenTracks_);
```

Comment:
This is cheap, but not free.
Could you move it back outside of the loop ?

Reply:
Sure, fixed in latest commits.